### ISSUE TITLE
LINKING YT ACCOUNT - Internal Server Error (There's been an error, and it's of the dubious sort.)

### ISSUE THREAD

**Issue by LuaCascade** [gamejolt/next-issue-tracker#1477]
![image](https://user-images.githubusercontent.com/69215033/115157753-f56c1700-a058-11eb-82fa-1e420564862f.png)

This shows up every single time I try to link my YT account to my profile...I'm not sure why it's happening but it's bothersome

**Comment by nilllzz** [comment]
Thanks for reporting, we are working on resolving this - we are waiting on a reply from YouTube currently.

**Comment by RevTapia** [comment]
the same thing happened I was going to the website to go check something it said the same thing I didn't do what you did but we don't know what going on?

**Comment by nilllzz** [comment]
> 
> 
> the same thing happened I was going to the website to go check something it said the same thing I didn't do what you did but we don't know what going on?

Does it still happen?

**Comment by RevTapia** [comment]
yep im still going on the website but it said there is a internal server error

**Comment by nilllzz** [comment]
> 
> 
> yep im still going on the website but it said there is a internal server error

Can you create a new issue and include a step-by-step of what you were doing to cause this internal server error.

**Comment by RevTapia** [comment]
will i did nothing but just looking I was doing things I went back then I see it can you guys fix it

**Comment by RevTapia** [comment]
pls fix it ;(

**Comment by RevTapia** [comment]
when are you guys gonna fix it
